.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
       - id: gitleaks
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

Comment:
There hasn't been any activity on that repo in 4 months.

Reply:
The mirror is not really working and it's not syncing newer v3.x tags because it sees v4.x as newer...but the maintainer keeps closing issues about it...

Reply:
Can we change the repo?

Reply:
Unfortunately we can't, they moved out all of the pre-commit config from `prettier/prettier` to this repo.

Reply:
Maybe we just bite the bullet and use the alpha version. Any issues and we revert.

Reply:
we can try it..it's not very promising that they haven't released a new v4 alpha in 4 months 😬